src/components/Filters/WhereOption/WhereOption.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { Text } from '../../typography';
+import { TextSizes } from '../../typography/Text/Text.enums';
+import { FlexContainer } from '../../FlexContainer';
+import { getBorderRadius, getColor, pxToRem } from '../../../utils/helpers';
+
+const Container = styled(FlexContainer)`
+  background: ${getColor('graphite3H')};
+  border-radius: ${getBorderRadius};
+  width: '100%';
```

Comment:
hm those quotes shouldn't be necessary

Reply:
oh, good catch! it turned out `width` doesn't have to be declared at all. removed